app/routes/photos/components/GalleryPictureModal.js

```diff
@@ -111,6 +111,16 @@ const RenderGalleryPicture = ({
   </div>
 );
 
+function Swipeable(props: {
```

Comment:
Why no arrow function? 😢

Reply:
If it how we do it for the rest of the project, I could change it. Yet I personally prefer the traditional function syntax for these types of functions.